docs/app-areas.md

```diff
@@ -0,0 +1,20 @@
+An autonomous economic agent (AEA) is an intelligent agent whose goal is generating economic value for its owner. It can represent machines, humans, or data. 
+
+There are five general application areas for Fetch.ai agents.
+
+* **Inhabitants**: agents paired with real world hardware devices such as drones, laptops, heat sensors, etc.
+* **Interfaces**: facilitation agents which provide the necessary API interfaces for interaction between old and new economic models.
+* **Pure software**: software agents living in the digital space that interact with inhabitant and interface agents and others.
+* **Digital data sales agents**: pure software agents that attach to data sources in the data marketplace.
```

Comment:
change `in the data marketplace` to `and sell it via the open economic framework`